app/experimenter/nimbus-ui/src/components/PageDesign/index.tsx

```diff
@@ -8,7 +8,11 @@ import AppLayoutWithExperiment from "../AppLayoutWithExperiment";
 
 const PageDesign: React.FunctionComponent<RouteComponentProps> = () => {
   return (
-    <AppLayoutWithExperiment title="Design" testId="PageDesign">
+    <AppLayoutWithExperiment
+      title="Design"
+      testId="PageDesign"
+      analysisRequired={true}
```

Comment:
Minor, not a blocker: I think you can just specify the prop (not need to set `={true}`)